docs/wiki/DeploymentFlow.md

```diff
@@ -30,6 +30,12 @@ Modules in this reference implementation must be deployed in the following order
 |   8   | Built-In and Custom Policy Assignments | Creates policy assignments to provide governance at scale.                                                                                                                                 | Management Groups, Log Analytics Workspace & Custom Policy Definitions | [infra-as-code/bicep/modules/policy/assignments/alzDefaults](https://github.com/Azure/ALZ-Bicep/tree/main/infra-as-code/bicep/modules/policy/assignments/alzDefaults) |
 |   9   | Corp Connected Spoke Network           | Creates Spoke networking infrastructure with Virtual Network Peering to support Hub & Spoke network topology.  Spoke subscriptions are used for deploying construction sets and workloads. | Management Groups, Hub Networking & Subscription for spoke networking  | [infra-as-code/bicep/modules/spokeNetworking](https://github.com/Azure/ALZ-Bicep/tree/main/infra-as-code/bicep/modules/spokeNetworking)                               |
 
+The following are optional modules used to deploy components according to the Azure Landing Zone conceptual architecture.
```

Comment:
Not so much optional as alternatives? I would prefer a wording/order which made it clear that it's either/or.

Reply:
good point, renaming this as "Other Deployment Scenarios". Starting with vWAN as a scenario, consisting of 2 starter modules which must be deployed in order.